### PR TITLE
Update grpc to v1.0.1-GA

### LINKED ISSUE #26506: TestHealth test hangs sometimes.
There is bunch of lines like:
```
19:13:29 time="2016-09-12T19:12:23.897547700Z" level=debug msg="Running health check..." 
19:13:29 time="2016-09-12T19:12:23.897737700Z" level=debug msg="starting exec command 56c254074d48e5056c45747ca3f8454946d96288ac0fba60cef88a448546626b in container 9003f6182173b76f15a14dcdfce18934483990377dfd316dfdef2aad5c32ff15" 
19:13:29 time="2016-09-12T19:12:23.899218000Z" level=debug msg="Health check taking too long" 
19:13:29 time="2016-09-12T19:12:23.899277700Z" level=debug msg="attach: stdout: begin" 
19:13:29 time="2016-09-12T19:12:23.899333700Z" level=debug msg="attach: stderr: begin" 
19:13:29 time="2016-09-12T19:12:23.899363900Z" level=debug msg="attach: stderr: end" 
19:13:29 time="2016-09-12T19:12:23.899379700Z" level=debug msg="attach: stdout: end" 
19:13:29 time="2016-09-12T19:12:23.900219800Z" level=warning msg="Health check error: rpc error: code = 4 desc = context deadline exceeded" 
```
It seems like it is related to latest containerd update in 6bc9a2d563ace9fa604de15731e263b353e83120

### PR DESCRIPTION
**Pull request by LK4D4**
**\- What I did**
Updated grpc, protobuf and gogoproto to stable versions.

**\- How I did it**
Vendored grpc, protobuf, gogoproto, containerd and swarmkit.

**\- How to verify it**
Run integration tests

**\- Description for the changelog**
Use grpc v1.0.1-GA

**\- A picture of a cute animal (not mandatory but encouraged)**
cat

This PR mostly show that it's possible to update grpc and we can merge swarmkit and containerd updates.
Fix #26506 
ping @aaronlehmann

### CODE REVIEW

**Comment by LK4D4** [comment]
Swarmkit part was merged.

**Comment by aaronlehmann** [comment]
@LK4D4: Can you update this to include containerd from the master branch?

**Comment by LK4D4** [comment]
@aaronlehmann Updated. removed WIP mark, this is ready for review

**Comment by LK4D4** [comment]
I'm not sure what vendor check wants

**Comment by LK4D4** [comment]
make validate works locally :(

**Comment by LK4D4** [comment]
I also need docker/swarmkit#1549
But let's see if TestHealth is fixed.

File: Dockerfile [hunk 1]
```diff
@@ -243,10 +243,10 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Install containerd
-ENV CONTAINERD_COMMIT 4eb3147efcc41a24ec5065795e06b0ec8e12565e
+ENV CONTAINERD_COMMIT 79225b971e97ea18252312f8573f48e88e3330c0
 RUN set -x \
 	&& export GOPATH="$(mktemp -d)" \
-	&& git clone https://github.com/docker/containerd.git "$GOPATH/src/github.com/docker/containerd" \
+	&& git clone https://github.com/mlaventure/containerd.git "$GOPATH/src/github.com/docker/containerd" \
```
Comment (vdemeester) [inline Dockerfile]: This is temporary, right ? :sweat_smile:

Comment (LK4D4) [inline Dockerfile]: Haha, we can only hope.

**Comment by LK4D4** [comment]
I think janky failure is some unrelated race in exec (we saw it in other tests).

**Comment by LK4D4** [comment]
Lol, everything is passed apart from janky.

**Comment by aaronlehmann** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM :frog: and all green :green_heart:

**Pull request merged**